addons/minio/templates/minio-scripts-template.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "minio.scriptsTplName" . }}
+  labels:
+    {{- include "minio.labels" . | nindent 4 }}
+data:
+  {{- with include "minio.extend.scripts" . }}
```

Comment:
Just use `{{ (.Files.Glob "scripts/*.sh").AsConfig | indent 2 }}`?

https://helm.sh/docs/chart_template_guide/accessing_files/#configmap-and-secrets-utility-functions

Reply:
> Just use `{{ (.Files.Glob "scripts/*.sh").AsConfig | indent 2 }}`?
> 
> https://helm.sh/docs/chart_template_guide/accessing_files/#configmap-and-secrets-utility-functions

done